packages/serverless-components/nextjs-cdk-construct/package.json

```diff
@@ -35,36 +35,16 @@
     "fs-extra": "9.1.0"
   },
   "peerDependencies": {
-    "@aws-cdk/aws-certificatemanager": "1.134.0",
-    "@aws-cdk/aws-cloudfront": "1.134.0",
-    "@aws-cdk/aws-cloudfront-origins": "1.134.0",
-    "@aws-cdk/aws-iam": "1.134.0",
-    "@aws-cdk/aws-lambda": "1.134.0",
-    "@aws-cdk/aws-lambda-event-sources": "1.134.0",
-    "@aws-cdk/aws-logs": "1.134.0",
-    "@aws-cdk/aws-route53": "1.134.0",
-    "@aws-cdk/aws-route53-targets": "1.134.0",
-    "@aws-cdk/aws-s3": "1.134.0",
-    "@aws-cdk/aws-s3-deployment": "1.134.0",
-    "@aws-cdk/aws-sqs": "1.134.0",
-    "@aws-cdk/core": "1.134.0"
+    "aws-cdk": "2.0.0",
+    "aws-cdk-lib": "2.0.0",
+    "constructs": "10.0.0"
```

Comment:
Sorry @n04h , I actually meant these package dependencies, not the dependencies of the example project which I commented about. I see those peer dependencies were pinned before your change. I think for TypeScript users it would make sense to allow patch (`~`) if not minor (`^`) version upgrades.